arcade/examples/minimap_camera.py

```diff
@@ -0,0 +1,212 @@
+"""
+Work with a mini-map
+
+Artwork from https://kenney.nl
+
+If Python and Arcade are installed, this example can be run from the command line with:
+python -m arcade.examples.minimap
```

Comment:
One of the errors appears to be because the run command does not match the name of the file exactly: Try this:
```suggestion
python -m arcade.examples.minimap_camera
```

Reply:
There is a minimap.py and now a minimap_camera.py. Both are correct

Reply:
I think there is a misunderstanding somewhere. To my knowledge, the tests expect the module name in an example's run instructions to match its filename exactly. This particular example (`minimap_camera.py`) is separate from `minimap.py`, so it must use the run instruction line I included above.

Reply:
Ok! I understand now!thanks